tools/flang1/flang1exe/accpp.c

```diff
@@ -2130,6 +2131,16 @@ _doincl(char *name, int type, LOGICAL include_next)
       goto found;
     }
 
+  if (type == 0) { /* could be absolute path, check where it leads to */
+    tmpfp = fopen(name, "r");
+    if (tmpfp) {
+      snprintf(fullname, MAX_PATHNAME_LEN, "%s", name);
+      idir.last = 0;
+      if (fclose(tmpfp) == 0)
```

Comment:
Using `fopen` and `fclose` here makes the preprocessor behave different for this specific case. In the other cases, `fndpath` calls `access` to determine if an included file exists. Shall we do the same here?